corehq/apps/app_manager/templates/app_manager/form_summary.html

```diff
@@ -81,46 +87,67 @@ <h5>
+                            <i data-bind="attr: {'class': 'fa-li fa text-muted fa-code-fork'}"></i>
+                            <span data-bind="text: form_filter"></span>
+                        </li>
+                    </ul>
+                    <!-- /ko -->
                     <ol data-bind="foreach: questions">
-                        <li data-bind="slideVisible: isVisible">
+                        <li data-bind="visible: isVisible">
                             <i data-bind="attr: { 'class': $root.questionIcon($data) }" title="type"></i>
-                            <span data-bind="visible: $root.showIds, text: value"></span>
-                            <span data-bind="visible: $root.showLabels, text: $root.translateQuestion($data)"></span>
-                            <span data-bind="visible: required" title="{% trans "This question is required"|escapejs %}">*</span>
+                            <!-- ko text: $root.showIds() ? value :  $root.translateQuestion($data) --><!-- /ko -->
+                            <!-- ko if: required -->
+                            <span title="{% trans "This question is required"|escapejs %}">*</span>
+                            <!-- /ko -->
+                            <!-- ko if: comment -->
                             <span class="text-muted" data-bind="visible: $root.showComments">
-                                &nbsp; <span data-bind="text: comment"></span>
+                                &nbsp; <!-- ko text: comment --><!-- /ko -->
                             </span>
-                            <!-- ko template: {name: 'question-attribute', data: { attribute: calculate, icon: 'fa-calculator',
-                                               visibleObs: $root.showCalculations } }--><!-- /ko -->
-                            <!-- ko template: {name: 'question-attribute', data: { attribute: relevant, icon: 'fa-code-fork',
-                                               visibleObs: $root.showRelevance } }--><!-- /ko -->
-                            <!-- ko template: {name: 'question-attribute', data: { attribute: constraint, icon: 'fa-ban',
-                                               visibleObs: $root.showConstraints } }--><!-- /ko -->
-                            <!-- ko template: {name: 'question-attribute', data: { attribute: setvalue, icon: 'fa-home',
-                                               visibleObs: $root.showDefaultValues } }--><!-- /ko -->
+                            <!-- /ko -->
+                            <!-- ko if:  calculate || relevant || constraint || setvalue  -->
+                            <ul class="fa-ul">
+                                <!-- ko if: calculate -->
+                                <li data-bind="visible: $root.showCalculations">
+                                    <i data-bind="attr: {'class': 'fa-li fa text-muted fa-calculator'}"></i>
+                                    <!-- ko text: calculate --><!-- /ko -->
```

Comment:
I haven't seen `text` attached to a comment-style knockout binding before. What does it turn into, a span?

Reply:
It doesn't turn into anything, it just inserts the text in that spot. 
![screenshot from 2019-01-21 17-52-29](https://user-images.githubusercontent.com/146896/51502735-5edf1b80-1da5-11e9-90a1-30258bdec38e.png)

Since there are already wrapping elements that hold the text, there's no need to have these extra span nodes.